04-networking.md

```diff
@@ -41,7 +41,7 @@ The following two resource groups will be created and populated with networking
 
    ```bash
    # [This takes less than one minute to run.]
-   az group create -n rg-enterprise-networking-hubs-$TEAM_NAME -l centralus
+   az group create -n rg-enterprise-networking-hubs-$TEAM_NAME -l $TEAM_LOCATION
```

Comment:
Are we using a central hub for all teams or creating a hub per team?  The directions above this line say central, but this creates a resource group for the team to create one.

Reply:
There will be individual hub per team. Typically hub could be in different location than spoke if we are planning to deploy multiple spokes across different regions but since we are doing 1 hub and 1 spoke for team, keeping it in same same team region for simplicity. 
Description mentions all hubs to be in `centrally-managed subscription`, will update description to be more clear